### DISCUSSION TITLE
手順書及びマニュアルの整備

### DISCUSSION THREAD

**Discussion by ghost** [su-its/ams-project#8] [General]
引継ぎ作業や新規顧客への導入手順書及び、エラー時のマニュアルや対処、その他エスカレーションなどを整備する必要がある。

システム的な部分ではなくあくまで組織論を主体に考えて頂けると幸いです。

(どうでも良いんですけど、導入手順書くらいはasciidocで良くね？)

**Comment by h-takeyeah** [comment]
> システム的な部分でなくあくまで組織論を主体に

ここはよくよく考えます
